sphinx/source/docs/releases/2.3.0.rst

```diff
@@ -152,3 +152,11 @@ This only affects extension creators. ``CartesianFrame`` is now responsible only
 for maintaining the coordinate system of a plot. Use the ``layout.center_panel``
 of a plot to access any layout-related capability. Note you can still use
 ``CartesianFrame.bbox``, so most code doesn't have to change.
+
+Figure sizing fixed
```

Comment:
Is this true for bothy Python and BokehJS? If this refers to BokehJS only, we should make that clear. For example: "BokehJS Figure sizing fixed" or "Figure sizing fixed in BokehJS".

Reply:
This is specific to BokehJS; header updated per your suggestion